qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -0,0 +1,30 @@
+- balloon_in_use:
+    type = driver_in_use
+    only Windows
+    extra_params += "-device virtio-balloon-pci,id=ballooning"
```

Comment:
framework has support balloon params to add balloon device, so please use it to setup your balloon device